.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
 venv/
-main/input/video_mp4
-main/output
\ No newline at end of file
+main/input/people_walking_mp4
+main/output
+object_detection/models
```

Comment:
add the following line:
`*.pyc`

to remove from this PR the following file:
`object_detection/__pycache__/tf_object_detection.cpython-36.pyc`